### ISSUE TITLE
Android x86 support

### ISSUE THREAD

**Issue by PaintballerAlex07** [cSploit/android#486]
Please add support for x86

**Comment by tonakriz** [comment]
Yes, I agree with you, we need it for x86, you can run it, but cant scan ports, cant inspect system, ...

**Comment by tux-mind** [comment]
I was pretty sure that was already an open TODO about x86....

However, it's already planned and we are still working on it, we're migrating the core from android-ndk to another build system ( see #220 )

thank you for your attention about this project, I hope to implement this feature soon, but we need more devs :grin:

**Comment by pahapoika** [comment]
@tux-mind i can be something like customer support guy. But i dont know much about android developing code so i cant help you more. But i can look for solves for peoples problems.

**Comment by pahapoika** [comment]
And i can do youtube videos that teachs how to use cSploit 2.
